modules/oe_content_person/oe_content_person.module

```diff
@@ -42,3 +46,102 @@ function oe_content_person_node_presave(EntityInterface $entity) {
     }
   }
 }
+
+/**
+ * Implements hook_inline_entity_form_entity_form_alter().
+ */
+function oe_content_person_inline_entity_form_entity_form_alter(array &$entity_form, FormStateInterface &$form_state) {
+  if ($entity_form['#entity_type'] !== 'oe_person_job') {
+    return;
+  }
+
+  // Set initial state when form is loaded.
+  $type = NULL;
+  $user_input = $form_state->getUserInput();
```

Comment:
I have not investigated why but oe_person_type value is absent in form state values in `oe_content_person_inline_entity_form_entity_form_alter()`.

<img width="771" alt="Screenshot 2021-02-25 at 11 33 49" src="https://user-images.githubusercontent.com/1183951/109125849-9d521c00-775d-11eb-8216-f27a43fbe685.png">

So user input is used. Exact conditions `if ($type !== 'eu')`, `if ($type !== 'non_eu')` are used to prevent form from fake date.